README.md

```diff
@@ -123,31 +121,32 @@ public static SootClass loadClass(String name, boolean main, String module) {
 }
 
 ```
-
+ModuleUtil.module_mode() helps you check if you have modules enabled in Soot. This is done based on whether the module path is set using the Options class.
 
 ### Example Configurations: Java 8, Java >= 9 Classpath, Java >= 9 Modulepath
 
 ```.java
 
-if(java < 9 ) {
+if(java < 9 ) { // when you want Java < 9 for Soot and you are not using Soot on modules
```

Comment:
Can we clarify? I think "when you have a target benchmark with Java < 9 and hence no modules" here.